src/zerobouncesdk/zb_find_email_response.py

```diff
@@ -0,0 +1,48 @@
+from typing import List
+
+from ._zb_response import ZBResponse
+
+
+class ZBDomainFormat(ZBResponse):
+    """This represents how a domain may format its emails
+    (and how likely it is to do so a certain way)
+    """
+
+    format: str = None
+
+    confidence: str = None
+
+
+class ZBFindEmailResponse(ZBResponse):
```

Comment:
is this recent?

because this one is:
![image](https://github.com/zerobounce/zero-bounce-python-sdk-setup/assets/128589427/ad0ed261-8904-41ec-a663-853dd01656dc)

Reply:
I think the api key has a fixed length. I changed a character from my key

Reply:
Handled the "Message" part.